src/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixer.php

```diff
@@ -82,6 +82,7 @@ final class PhpUnitTestCaseStaticMethodCallsFixer extends AbstractFixer implemen
         'assertClassNotHasAttribute' => true,
         'assertClassNotHasStaticAttribute' => true,
         'assertContains' => true,
+        'assertContainsEquals' => true,
```

Comment:
other PR, perhaps ?
like, it's nice to see this change in changelog, without dedidacted PR, it would not be visible

Reply:
Ok, I've reopened https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4337